src/app/content/components/BookBanner.tsx

```diff
@@ -248,6 +255,7 @@ export class BookBanner extends Component<PropTypes, {scrollTransition: boolean}
           onClick={(e: React.MouseEvent<HTMLAnchorElement>) => {
             this.handleLinkClick(e, bookUrl);
           }}
+          tabIndex={this.state.tabbableBanner !== 'big' ? -1 : undefined}
```

Comment:
I think it would be easier to read if this would be `===` check:
`this.state.tabbableBanner === 'mini' ? -1 : undefined`